src/sqlancer/stonedb/StoneDBProvider.java

```diff
@@ -42,7 +41,7 @@ protected StoneDBSchema readSchema() throws Exception {
     enum Action implements AbstractAction<StoneDBGlobalState> {
         SHOW_TABLES((g) -> new SQLQueryAdapter("SHOW TABLES")), //
         ALTER_TABLE(StoneDBTableAlterGenerator::generate), //
-        DELETE(StoneDBDeleteGenerator::generate), //
+        DELETE(sqlancer.stonedb.gen.StoneDBTableDeleteGenerator::generate), //
```

Comment:
remove blank comment lines

Reply:
Blank comment lines are used to place each action on each line. If removed, two actions will be placed on one line by the formatter, which I think is not as good as each action on one line?